### ISSUE TITLE
Change Explore Screen to Discover Screen

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#123]
Based on 4 non-visual interviews @Matthew-Moore240 and @terracoda decided that we need to make it more obvious that the ratio challenges are something learners have to "discover" rather than "explore".

Of course, they are exploring to discover, and we feel "discover" is a better word to name the screen.

This is the first of several issues that will support and reflect this design change from "explore" to "discover" ratio challenges.

Please be sure to change the buttons on the home screen and in the navigation bar.

**Comment by zepumph** [comment]
Tagging @emily-phet and @BLFiedler as this is a rename of the entire screen. I hope to get to this next week.

**Comment by terracoda** [comment]
I am assigning @emily-phet and @BLFiedler because we all want to move through these issues quickly this week.

@emily-phet and @BLFiedler please approve this change. It is essential to the design of the non-visual experience.

**Comment by terracoda** [comment]
This change would mean that the H1 and the first paragrpah of the screen summary would change to:

H1: {{Discover}} Screen, Ratio and Proportion 

P1: The interactive {{Discover Screen}} changes as you play with it. It has a Play Area and a Control Area.

other changes to the screen summaries are outlined in a different issue.

**Comment by BLFiedler** [comment]
Sounds good to me.

**Comment by emily-phet** [comment]
Generally I personally try to avoid "discover", as it carries with it something that feels presumptive to me, like the learner can or should discover something, when it's possible they can't or don't. 

But this sim is certainly unique from ones I've worked on in the past - they really do need to focus on figuring out what the challenges are. If it's important to @terracoda and @Matthew-Moore240 - "discover" it is!

**Comment by terracoda** [comment]
Thanks @emily-phet and @BLFiedler!

@Matthew-Moore240 and I do indeed like "discover", and we thought it would make sense to rename the screen. That said, I am **willing to hold off on actually renaming the screen**, and, to focus on strategically using the word "discover" for scaffolding in the rest of the State Descriptions.

I was all gung-ho to rename the screen and then I had 2 hesitations:
1. Renaming the screen, might actually crate a bunch of busy-work for @zepumph 
2. The word "Explore" is still relevant to the sim, so it is not like we won't use it, too.

I think we can move forward **with NOT immediately changing the screen name**. I would like to discuss this more in a design meeting next Tuesday, AFTER we have all other changes to the State Descriptions implemented.

I hope that sounds good to everyone, especially @zepumph.

**Comment by BLFiedler** [comment]
> But this sim is certainly unique from ones I've worked on in the past - they really do need to focus on figuring out what the challenges are. If it's important to @terracoda and @Matthew-Moore240 - "discover" it is!

I'll back this up and say I do think this sim really is a bit more unique both in it's abstraction and task-oriented nature. There is something they are trying to uncover that is withheld from them (and they are challenged to do so) until they put in the effort to find it, and while they can do that by "exploring" around, it also doesn't carry the weight with it that maybe they should in fact be looking for something in particular.

So, yeah, still good with Discover.

**Comment by zepumph** [comment]
This won't be very difficult, likely only 30 minutes of work or so. I'm happy to do it whenever, just assign me.

**Comment by terracoda** [comment]
Ok, well then, along with all the other issues that I have already posted for design changes for the State Descriptions, please make the screen name change, too. Assigning to you @zepumph.

**Comment by terracoda** [comment]
@zepumph, in terms of priorties, you can focus on the other changes first. Then we can have another read through before we change the screen name.

Sound good?

**Comment by zepumph** [comment]
Hey, I was about to start working on this, then I saw https://github.com/phetsims/ratio-and-proportion/issues/123#issuecomment-674123205. Do you want to do a read through before signing off on this change?

**Comment by terracoda** [comment]
@zepumph, reading through the current A11y View now :-)

**Comment by terracoda** [comment]
@zepumph, yep, I think we can go ahead with changing the Explore Screen to the Discovery Screen.

**Comment by zepumph** [comment]
I just made a blanket rename from explore->discover in the sim. I am about 90% sure that there is a usage that I shouldn't have renamed, especially in PDOM description. @terracoda could you please look at every usage of "discover" (case insensitive) in https://github.com/phetsims/ratio-and-proportion/blob/master/ratio-and-proportion-strings_en.json and either change them back to explore or let me know and I can do it. Sorry for the hassle, this is just sorta how it happened.

**Comment by zepumph** [comment]
I see 7 usages in the strings file that should be looked at.

**Comment by zepumph** [comment]
I will take a look at the above usages to ready the dev version for interviews tomorrow.

**Comment by zepumph** [comment]
I changed most PDOM strings back to use "explore" instead of "discover". @Matthew-Moore240 please review the above changes, and let me know if any usages of "explore" should become "discover.

**Comment by Matthew-Moore240** [comment]
I couldnt find any that felt weird so it looks good to me!

**Comment by Matthew-Moore240** [comment]
Ill go ahead and close this and if anything pops back up we can just create a new issue.
